_includes/index-categories.html

```diff
@@ -28,7 +28,13 @@ <h2 id="{{ cat.id }}">{{ cat.name }} <small>[{{ cat.id }}]</small></a></h2>
     {% for page in site.pages %}
     <ul>
       {% if page.categories contains cat.id %}
-      <li><a href="{{ site.baseurl }}{{ page.url }}">{{ page.title }}</a></li>
+      <li><a href="{{ site.baseurl }}{{ page.url }}">
+        {% if page.stability-level != "default" or page.stability-level != "community" %}
```

Comment:
@bstansberry The generated proposal will not add prefixes to their title if they are at either the `community` or `default` stability level.

Is that ok or do you want to display one or the other?

Reply:
@jmesnil We should display community.

Reply:
done (both in the proposal pages and the index page)